### PR TITLE
Hide flutter run_mojo

### PR DESCRIPTION
**Pull request by eseidelGoogle**
Someone asked about this command in #flutter which made me realize
that it would just be confusing to most current consumers of flutter
so lets hide this for now.

@devoncarew

### CODE REVIEW

File: packages/flutter_tools/lib/src/commands/run_mojo.dart [hunk 1]
```diff
@@ -19,9 +19,10 @@ const String _kDefaultBundlePath = 'build/app.flx';
 
 class RunMojoCommand extends FlutterCommand {
   final String name = 'run_mojo';
-  final String description = 'Run a Flutter app in mojo.';
+  final String description = 'Run a Flutter app in mojo (from github.com/domokit/mojo).';
+  final bool hideCommand;
```
Comment (abarth) [inline packages/flutter_tools/lib/src/commands/run_mojo.dart]: hideCommand -> _hideCommand

**Comment by abarth** [comment]
LGTM

File: packages/flutter_tools/lib/src/commands/run_mojo.dart (hunk 1 above)
Comment (devoncarew) [inline packages/flutter_tools/lib/src/commands/run_mojo.dart]: He'll need to change the ctor to something like:
```
RunMojoCommand({ bool hideCommand: false }) : _hideCommand = hideCommand {
...
}
```
which is a bit more verbose.

Could also do:
```
final bool hidden;

RunMojoCommand({ this.hidden: false }) {
...
}
```

which would override the getter from the parent Command class.

**Comment by devoncarew** [comment]
lgtm also. No particular preference re: the ctor. We should make this and the daemon command agree at some point if this one changes.

**Comment by eseidelGoogle** [comment]
done!

**Pull request merged**